### PR TITLE
Modifications necessary to kerberize guacamole

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

This pull request makes modifications necessary to kerberize guacamole.  These include:
* Adding tasks for configure httpd for PAM and Kerberos
* Adding httpd configurations for regular users as well as administrative users
* Changing the path to the SSL certificate files to reflect the use of httpd instead of nginx
* Adding a script to set up the `guacamole` and `guacamole-admin` FreeIPA services

See also:
* cisagov/guacamole-composition#9
* cisagov/ansible-role-httpd#2
* cisagov/cool-assessment-terraform#58

## 💭 Motivation and Context

Guacamole must be kerberized in the COOL.

## 🧪 Testing

I used these changes to successfully a kerberized version of guacamole to env0 in our staging environment.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: files/guacamole_httpd_config [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        GssapiUseSessions On
+        Session on
+        SessionCookieName gssapi_session path=/;httponly;secure;
+
+        require pam-account guacamole
+
+        ProxyPass http://localhost/guacamole/
+        ProxyPassReverse http://localhost/guacamole/
+
+        RequestHeader set REMOTE_USER guacuser
```
Comment (dav3r) [inline files/guacamole_httpd_config:33]: The magic sauce! ⚡

File: meta/main.yml [hunk 2]
```diff
@@ -10,9 +10,23 @@ galaxy_info:
...
+        - bionic
   galaxy_tags:
     - guacamole
 
 dependencies:
   - src: https://github.com/cisagov/ansible-role-docker
     name: docker
+  - src: https://github.com/cisagov/ansible-role-httpd
+    name: httpd
+    version: feature/kerberize
```
Comment (dav3r) [inline meta/main.yml]: 🎗️ You know the drill.

File: molecule/default/requirements.yml [hunk 3]
```diff
@@ -1,5 +1,8 @@
 ---
 - src: https://github.com/cisagov/ansible-role-docker
   name: docker
+- src: https://github.com/cisagov/ansible-role-httpd
+  name: httpd
+  version: feature/kerberize
```
Comment (dav3r) [inline molecule/default/requirements.yml]: 🎗️ Here too.

File: tasks/main.yml [hunk 4]
```diff
@@ -9,7 +9,7 @@
 - name: Download and untar the guacamole-composition tarball
   unarchive:
     src: >
-      https://api.github.com/repos/cisagov/guacamole-composition/tarball/develop
+      https://api.github.com/repos/cisagov/guacamole-composition/tarball/feature/kerberization
```
Comment (dav3r) [inline tasks/main.yml]: 🎗️

**Review by dav3r — APPROVED** [review]
Excellent work! 🥮

File: molecule/default/tests/test_default.py [hunk 5]
```diff
@@ -12,7 +12,57 @@
...
+def test_files(host, f, perms):
+    """Test that the expected files were created and are non-empty."""
+    assert host.file(f).exists
+    assert host.file(f).is_file
+    assert host.file(f).content
+    assert host.file(f).mode == perms
+
+
+@pytest.mark.parametrize(
+    "link, target",
```
Comment (felddy) [inline molecule/default/tests/test_default.py:44]: Nice!  💪

**Review by felddy — APPROVED** [review]
pug_dribbble

File: requirements.txt [hunk 6]
```diff
@@ -1 +1,3 @@
 boto3
```
Comment (mcdonnnj) [inline requirements.txt]: Is this requirement still needed? It looks like in #5 all of the AWS related fu was removed, but there appear to be lingering elements like the `terraform` directory and this requirement. You've also removed the `boto3` installation performed in `molecule/default/prepare.yml`.

**Review by mcdonnnj — APPROVED** [review]
Some really strong work here 💪. I had one question below.

File: requirements.txt (hunk 6 above)
Comment (jsf9k) [reply requirements.txt]: I think you are correct once again.  I will test and confirm.

Comment (jsf9k) [reply requirements.txt]: See commits 3e5a3f1 and e70dfc1.

Comment (mcdonnnj) [reply requirements.txt]: Nice 👍 Thanks for checking.

File: tasks/main.yml (hunk 4 above)
Comment (jsf9k) [reply tasks/main.yml]: See commit 31a755c.

File: meta/main.yml (hunk 2 above)
Comment (jsf9k) [reply meta/main.yml]: See commit 4da20df.

File: molecule/default/requirements.yml (hunk 3 above)
Comment (jsf9k) [reply molecule/default/requirements.yml]: See commit 4da20df.

**Pull request merged**